XenModel/Messages.resx

```diff
@@ -5932,6 +5935,12 @@ Would you like to eject these ISOs before continuing?</value>
   <data name="FILE_ALL" xml:space="preserve">
     <value>All files</value>
   </data>
+  <data name="FILE_NAME_IS_DEVICE_NAME_ERROR_MESSAGE" xml:space="preserve">
+    <value>A file name can not be a device name.</value>

```

Comment:
"can not" is still here.

Reply:
oh silly me, changed the cannot to can not.....oops meant to go the other way